include/cutlass/gemm/device/gemm_universal_adapter.h

```diff
@@ -410,14 +410,16 @@ class GemmUniversalAdapter<
         const auto sycl_block = syclcompat::dim3(block.x, block.y, block.z);
         const auto sycl_grid = syclcompat::dim3(grid.x, grid.y, grid.z);
 
-#if defined (SYCL_INTEL_TARGET)
         using namespace syclcompat::experimental;
+#if defined (SYCL_INTEL_TARGET)
         auto event = launch<device_kernel<GemmKernel>>(launch_policy{
           sycl_grid, sycl_block, local_mem_size{static_cast<std::size_t>(smem_size)}, 
           kernel_properties{sycl_exp::sub_group_size<DispatchPolicy::SubgroupSize>}
         }, params);
 #else
-        auto event = syclcompat::launch<device_kernel<GemmKernel>>(sycl_grid, sycl_block, smem_size, params);
```

Comment:
Isn't this launcher still available at https://github.com/intel/llvm/blob/06a3902964d0fdcb20131cbf182184db2c1fd2b6/sycl/include/syclcompat/launch.hpp#L117?

Reply:
That one is still available. We are missing the one with the smem_size argument (removed here https://github.com/intel/llvm/pull/14441/files#diff-731a626bb260c2643c8ba82151630ad99e245ffa3fdfe924db4fdd4b53ed3557)

Reply:
🤦🏻 🙏🏻